ffi/examples/read-table/CMakeLists.txt

```diff
@@ -26,6 +26,8 @@ if(MSVC)
 else()
   # no-strict-prototypes because arrow headers have fn defs without prototypes
   target_compile_options(read_table PRIVATE -Wall -Wextra -Wpedantic -Werror -Wno-strict-prototypes)
+  target_compile_options(read_table PRIVATE -fsanitize=address,leak)
```

Comment:
will revert, just want to see that it reports no error in ffi

Reply:
We shouldn't need this? `address` includes `leak`, if leak detection is supported -- which is why I had to disable leak detection directly in the [C code](https://github.com/delta-incubator/delta-kernel-rs/blob/main/ffi/examples/read-table/read_table.c#L9-L11).